### PR TITLE
Add "Parallel Select (Anime)"

### PR DESCRIPTION
**Pull request by idk-what-to-name-here**
https://yugipedia.com/wiki/Parallel_Select_(anime)

- [X] I am following the [contributing guidelines](https://github.com/ProjectIgnis/CardScripts/blob/master/CONTRIBUTING.md).
- [X] I am making the [changes to modernize the scripts](https://github.com/ProjectIgnis/CardScripts/blob/master/MODERNIZING.md).

**New card checklist**

- [X] I have submitted a corresponding database entry for [BabelCDB](https://github.com/ProjectIgnis/BabelCDB/blob/master/README.md) according to its guidelines. Link to the pull request: _https://github.com/larry126/BabelCDB/pull/126_
- [X] This card is not marked with a reason to not be added in the unofficial card tracker in `#card-scripting-101`.
Supervising staff member(s): _@larry126_

### CODE REVIEW

**Review by larry126 — CHANGES_REQUESTED** [review]

**Comment by idk-what-to-name-here** [comment]
done

File: unofficial/c511027651.lua [hunk 1]
```diff
@@ -0,0 +1,36 @@
+local s,id=GetID()
+function s.initial_effect(c)
+	--Activate
+	local e1=Effect.CreateEffect(c)
+	e1:SetType(EFFECT_TYPE_ACTIVATE)
+	e1:SetCategory(CATEGORY_DRAW)
```
Comment (larry126) [inline unofficial/c511027651.lua:6]: You still need CATEGORY_TOHAND since you are adding something to your hand other than via drawing

**Review by larry126 — CHANGES_REQUESTED** [review]
And where have the card names gone? xD

**Comment by idk-what-to-name-here** [comment]
done

**Review by larry126 — APPROVED** [review]

**Pull request merged**